apis/atmos/src/main/java/org/jclouds/atmos/xml/ListDirectoryResponseHandler.java

```diff
@@ -49,11 +53,17 @@ public void endElement(String uri, String name, String qName) {
       } else if (qName.equals("FileType")) {
          currentType = FileType.fromValue(currentText.toString().trim());
       } else if (qName.equals("Filename")) {
+         currentFileName = currentText.toString().trim();
+         if (currentFileName.equals(""))
+            currentFileName = null;
+      } else if (qName.equals("Name")) {
          currentName = currentText.toString().trim();
-         if (currentName.equals(""))
-            currentName = null;
+      } else if (qName.equals("Value")) {
+         if (currentName.equals("size")) {
```

Comment:
I cannot find sufficient mention of case-sensitivity in the Atmos documentation to motivate this change.  If you can find additional references we can change the entire provider to this scheme.